src/resources/Namespaces/AllNamespacesDetails.js

```diff
@@ -28,7 +28,7 @@ export function AllNamespacesDetails(props) {
         title={t('navigation.all-namespaces')}
         breadcrumbItems={breadcrumbItems}
         content={
-          <div className="panel-grid">
+          <div className="panel-grid bsl-margin-top--md">
```

Comment:
Please import spacing from UI5 and make it like this
```suggestion
          <div className="panel-grid" style={spacing.sapUiMediumMarginTop}>
```